cpp/core/utils/Print.h

```diff
@@ -12,7 +12,7 @@ namespace gluten {
 
 // just for test, will be deleted in the future
 #ifndef GLUTEN_PRINT_DEBUG
-// #define GLUTEN_PRINT_DEBUG
+#define GLUTEN_PRINT_DEBUG
```

Comment:
Is this the final version? if is, please remove this.

Reply:
ok, I will delete it